internal/gapicgen/git/git.go

```diff
@@ -107,17 +109,18 @@ func ParseChangeInfo(googleapisDir string, hashes []string, gapicPkgs map[string
 		if err != nil {
 			return nil, err
 		}
-		var pkg string
+		var pkg, pkgDir string
 		for _, file := range files {
 			ss := strings.Split(file, "/")
 			if len(ss) == 0 {
 				continue
 			}
 			// remove filename from path
 			strings.Join(ss[:len(ss)-1], "/")
-			tempPkg := gapicPkgs[strings.Join(ss[:len(ss)-1], "/")]
-			if tempPkg != "" {
-				pkg = tempPkg
+			importPath := gapicPkgs[strings.Join(ss[:len(ss)-1], "/")]
```

Comment:
Remove `ss` & spurious line and use https://golang.org/pkg/path/filepath/#Dir instead.

Reply:
Done.